src/test/Square.test.js

```diff
@@ -1,17 +1,42 @@
 import React from "react";
-import Enzyme, {mount} from "enzyme";
-import App from "../App"
+import Enzyme, { mount } from "enzyme";
+import Square from "../components/Square";
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
 
-Enzyme.configure({ adapter: New Adapter() });
+Enzyme.configure({ adapter: new Adapter() });
 
-describe("<Square />", ()=>{
-    it("render a blank <Square /> component",()=>{
-
-    })
-    it("render a <Square /> with an X", ()=>{
-
-    })
-    it("render a <Square /> with an O",()=>{
-
-    })
-})
\ No newline at end of file
+describe("<Square />", () => {
+  it("render a blank <Square /> component", () => {
+    const imgSrc = "blank/img.png";
+    const wrapper = mount(<Square src={imgSrc} />);
+    console.log(wrapper.debug());
+    const props = wrapper.props();
+    expect(props.src).toBe("blank/img.png");
+  });
+  it("render a <Square /> with an X", () => {
+    const imgSrc = "X/img.png";
+    const wrapper = mount(<Square src={imgSrc} />);
+    const props = wrapper.props();
+    expect(props.src).toBe("X/img.png");
+  });
+  it("render a <Square /> with an O", () => {
+    const imgSrc = "O/img.png";
+    const wrapper = mount(<Square src={imgSrc} />);
+    const props = wrapper.props();
+    expect(props.src).toBe("O/img.png");
+  });
+  it("disables the square when the square is marked", () => {
+    const imgSrc = "blank/img.png";
+    const disabled = true;
+    const wrapper = mount(<Square src={imgSrc} disabled={disabled} />);
+    const props = wrapper.props();
+    expect(props.disabled).toBe(true);
```

Comment:
I would imagine this test would also show the square is marked.

Reply:
If the game is over the square can be unmarked and disabled

Reply:
Yes. I will make the correct changes.